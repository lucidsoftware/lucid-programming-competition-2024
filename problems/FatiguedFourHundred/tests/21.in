version https://git-lfs.github.com/spec/v1
oid sha256:b73d29be9bdba362a3f25069c7e98c585f120741cc5863213da263995ca028dc
size 482543
