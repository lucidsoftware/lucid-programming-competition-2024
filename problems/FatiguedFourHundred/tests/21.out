version https://git-lfs.github.com/spec/v1
oid sha256:bf0985939ea2fa094f0583001abc989a121f746ccf5585676fd7985e8db6e731
size 356457
